docs/pages/project/contributing.md

```diff
@@ -3,13 +3,26 @@ layout: page
 title: Contributing
 permalink: project/contributing
 ---
-
 # <a name="contributing">Contributing Overview</a>
-Please do! Thanks for your help improving the project! :balloon:
 
-All contributors are welcome. Please see the [newcomers welcome guide](https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit) for how, where and why to contribute. This project is community-built and welcomes collaboration. Contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct.
+Please do! Thanks for your help! :balloon:
+
+This project is community-built and welcomes collaboration. Contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) 
```

Comment:
Not an issue that you created, but the reference to the Contributor Covenant is a residual artifact from before we switched to adhering to the CNCF's Code of Conduct. Could you update this while you're in here?

Reply:
Sure. Would this [link](https://events.linuxfoundation.org/about/code-of-conduct/) be appropriate for the Code of Conduct?